pages/settings/SettingsLayout.qml

```diff
@@ -58,6 +58,14 @@ Rectangle {
             text: qsTr("Custom decorations") + translationManager.emptyString
         }
 
+        MoneroComponents.CheckBox {
+            id: checkForUpdatesCheckBox
+            enabled: !disableCheckUpdatesFlag
+            checked: persistentSettings.checkForUpdates && !disableCheckUpdatesFlag
+            onClicked: persistentSettings.checkForUpdates
```

Comment:
.

Reply:
yea not sure what happened there, updated